q

```diff
@@ -0,0 +1,93 @@
+--- python/gui/auto_generated/codeeditors/qgscodeeditor.sip.in	2022-10-22 16:00:09.183171185 +1000
```

Comment:
What's this q file? The machine is broken :fearful: :wink:

Reply:
oops, accidental commit!